docs/docs/use-global.md

```diff
@@ -0,0 +1,92 @@
+---
+id: use-global
+title: The useGlobal hook
+---
+
+Croods has a global state manager that can be of use if there's need for manipulating globally available state. 
+
+With it you can save and request information from this store without needing to tamper with Croods store manually, **which is highly disencouraged**.
+
+## Usage
+
+### First, you set up your global state file:
+
+- import `useStore` hook
+- setup an initial state
+- setup actions that interact with the state
+- export using the hook
+
+```js
+// src/useGlobal.js
+
+import { useStore } from 'croods'
+
+const initialState = {
+  menuOpen: false, // example
+  searchTerm: undefined,
+}
+
+const actions = {
+  toggleMenu: (store) => { // store is always passed as first parameter
+    store.setState({ menuOpen: !store.state.menuOpen })
+  },
+  doSearch: (store, searchTerm) => { // second parameter is received by useGlobal hook
```

Comment:
```suggestion
  doSearch: (store, searchTerm) => { // the second and following parameters are the ones you'll send when using this action
```

Reply:
Not sure it is the best way to put it, but I think it was a little abstract... any ideas on how to improve this line?